wire/message.go

```diff
@@ -43,6 +43,7 @@ const (
 	CmdInvTransaction
 	CmdRequestTransactions
 	CmdIBDBlock
+	CmdNotFound
```

Comment:
Please change (here and in stringer) to "CmdTransactionNotFound" (and in stringer too)